Taskfile.yml

```diff
@@ -51,25 +49,29 @@ tasks:
 
   compile:fe:
     desc: Runs less, pug and node
-    dir: public
+    dir: web
     sources:
-      - css/semaphore.less
-      - html/*.pug
-      - html/**/*.pug
+      - resources/css/semaphore.less
+      - resources/html/*.pug
+      - resources/html/**/*.pug
     generates:
-      - css/semaphore.css
-      - html/*.html
-      - html/**/*.html
-      - js/bundle.json
-    cmds:
-      - lessc css/semaphore.less > css/semaphore.css
-      - pug $(find ./html/ -name "*.pug")
-      - node ./bundler.js
+      - public/css/semaphore.css
+      - public/html/*.html
+      - public/html/**/*.html
+      - bundle.json
+    cmds:
+      - mkdir -p public/css
+      - mkdir -p public/html
+      - mkdir -p public/fonts
+      - lessc resources/less/semaphore.less > public/css/semaphore.css
+      - pug resources/pug --out public/html
+      - cp ../node_modules/font-awesome/fonts/* public/fonts
+      - node bundler.js
 
   compile:be:
     desc: Runs Packr for static assets
     sources:
-      - public/*
+      - web/resources/*
```

Comment:
this isnt going to work because packr wants to pack the final set of assets, ie the public folder and not what we build it from

Reply:
Fixed